app/src/main/java/transcendence/org/transcendencewallet/service/TranscendenceWalletService.java

```diff
@@ -328,7 +379,10 @@ public void onCreate() {
             final String lockName = getPackageName() + " blockchain sync";
             final PowerManager pm = (PowerManager) getSystemService(Context.POWER_SERVICE);
             wakeLock = pm.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, lockName);
-            nm = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+            createNotificationChannel();
+            startServiceForeground();
```

Comment:
It is better to describe what is the magic with start and stop sequentially

Reply:
Done